eng/pipelines/templates/steps/build-test.yml

```diff
@@ -105,13 +105,19 @@ steps:
     condition: and(succeeded(), ${{ parameters.RunCoverage }})
 
   - task: PublishPipelineArtifact@1
-    displayName: 'Publish .coverage files'
+    displayName: 'Publish raw .coverage files'
     continueOnError: true
     condition: and(succeeded(), ${{ parameters.RunCoverage }})
     inputs:
       targetPath: '$(Build.SourcesDirectory)/_coverage'
       artifactType: 'pipeline'
 
+  - pwsh: |
+      python -m pip install coverage==4.5.4
```

Comment:
@mccoyp this will stay for now, but we should bump `pytest-cov`, `coverage`, etc across the board (which will remove the need for this).

I don't want to break into jail on that work item here, as I want this to be enabled for our PRs sooner rather than later.

Reply:
Here is the follow-up issue. https://github.com/Azure/azure-sdk-for-python/issues/27828

Won't just get forgotten.